04-data-structures-part1.Rmd

```diff
@@ -339,6 +339,16 @@ In this case, we've explicitly told R that "control" should represented by 1, an
 "case" by 2. This designation can be very important for interpreting the
 results of statistical models!
 
+> ## Challenge 5 {.challenge}
+> Having to factors:
+> ```{r, eval=FALSE}
+> sex <- factor(c("M", "F", "F", "M", "M", "F"))
+> age <- factor(sample(100, size=6))
+>```
+> Can you extract values stored by those factors?
```

Comment:
It is unclear to me if you are requesting the values ```as.numeric(sex)``` or the labelled values ```sex```. Perhaps asking, "what value has been assigned to each label" makes the relationship more explicit.